reana_workflow_controller/consumer.py

```diff
@@ -144,7 +144,7 @@ def _update_run_progress(workflow_uuid, msg):
     job_progress = workflow.job_progress
     if "cached" in msg["progress"]:
         cached_jobs = msg["progress"]["cached"]  # noqa: F841
-    for status in PROGRESS_STATUSES:
+    for status, _ in PROGRESS_STATUSES:
```

Comment:
Minor: We can keep `job_status` in place of `_`.

Reply:
Why? the second value of the tuple is not used, so for clarity, I remove any importance by assigning it to `_`. The rest remains unchanged.